cpp/log/cert.cc

```diff
@@ -887,10 +963,23 @@ bool ValidateRedactionSubjectAltNames(STACK_OF(GENERAL_NAME) *
 }  // namespace
 
 
+util::Status Cert::SubjectAltNames(vector<string>* dns_alt_names) const {
+  ScopedGENERAL_NAMEStack subject_alt_names(
+      static_cast<STACK_OF(GENERAL_NAME)*>(X509_get_ext_d2i(
+          x509_.get(), NID_subject_alt_name, nullptr, nullptr)));
+
+  if (subject_alt_names) {
```

Comment:
Could skip the conditional, since it duplicates the check.

Also, that helper method isn't very big, could just be inlined? Or is it going to be used again in some upcoming PR?